diff --git a/Telegram/lib_ui/ui/rp_widget.cpp b/Telegram/lib_ui/ui/rp_widget.cpp
index 05cb414..d056557 100644
--- a/Telegram/lib_ui/ui/rp_widget.cpp
+++ b/Telegram/lib_ui/ui/rp_widget.cpp
@@ -19,7 +19,8 @@ TWidget::TWidget(QWidget *parent)
 		auto format = QSurfaceFormat::defaultFormat();
 		format.setSwapInterval(0);
 #ifdef Q_OS_MAC
-		format.setColorSpace(QColorSpace::SRgb);
+		auto const colorSpace = QColorSpace::SRgb;
+		format.setColorSpace(colorSpace);
 #endif // Q_OS_MAC
 		QSurfaceFormat::setDefaultFormat(format);
 		return true;
