diff --git a/lib/xdg/com.ayugram.desktop.desktop b/lib/xdg/com.ayugram.desktop.desktop
index 3c5f7bde0..e78454cf1 100644
--- a/lib/xdg/com.ayugram.desktop.desktop
+++ b/lib/xdg/com.ayugram.desktop.desktop
@@ -2,8 +2,8 @@
 Name=AyuGram Desktop
 Comment=Desktop version of AyuGram - ToS breaking Telegram client
 TryExec=ayugram-desktop
-Exec=DESKTOPINTEGRATION=1 ayugram-desktop -- %u
-Icon=ayugram
+Exec=ayugram-desktop -- %u
+Icon=share/icons/hicolor/64x64/apps/ayugram.png
 Terminal=false
 StartupWMClass=AyuGram
 Type=Application
