diff --git a/Telegram/SourceFiles/platform/mac/main_window_mac.h b/Telegram/SourceFiles/platform/mac/main_window_mac.h
index 63ed4009b..b8090c426 100644
--- a/Telegram/SourceFiles/platform/mac/main_window_mac.h
+++ b/Telegram/SourceFiles/platform/mac/main_window_mac.h
@@ -49,10 +49,7 @@ protected:
 private:
 	friend class Private;
 
-	bool nativeEvent(
-		const QByteArray &eventType,
-		void *message,
-		qintptr *result) override;
+    //	bool nativeEvent(const QByteArray &eventType, void *message, qintptr *result) override;
 
 	void hideAndDeactivate();
 	void updateDockCounter();
