diff --git a/Telegram/SourceFiles/platform/mac/main_window_mac.h b/Telegram/SourceFiles/platform/mac/main_window_mac.h
index 63ed4009b..b8090c426 100644
--- a/Telegram/SourceFiles/platform/mac/main_window_mac.h
+++ b/Telegram/SourceFiles/platform/mac/main_window_mac.h
@@ -49,10 +49,7 @@ protected:
 private:
 	friend class Private;
 
-	bool nativeEvent(
-		const QByteArray &eventType,
-		void *message,
-		qintptr *result) override;
+    //	bool nativeEvent(const QByteArray &eventType, void *message, qintptr *result) override;
 
 	void hideAndDeactivate();
 	void updateDockCounter();
diff --git a/Telegram/lib_ui b/Telegram/lib_ui
--- a/Telegram/lib_ui
+++ b/Telegram/lib_ui
@@ -1 +1 @@
-Subproject commit 563092d3920fe1e6a6c326e25d4abb92f33fa36f
+Subproject commit 563092d3920fe1e6a6c326e25d4abb92f33fa36f-dirty
